x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/styles.ts

```diff
@@ -143,7 +143,11 @@ export const WaterfallChartLegendContainer = euiStyled.div`
   box-shadow: 0px -1px 4px 0px ${(props) => props.theme.eui.euiColorLightShade};
 `; // NOTE: EuiShadowColor is a little too dark to work with the background-color
 
-export const WaterfallChartTooltip = euiStyled.div`
+export const WaterfallResponsiveMaxWidth = euiStyled.div`
```

Comment:
I feel like this name is a little bit generic. Feels like it may be intended to use in multiple places when this is really just meant to be used for the tooltip. I suppose it could be reused, but I'm not sure as of yet what we're reuse it for.

Reply:
Yeah I was unsure what to name it, can revise to `WaterfallTooltipResponsiveMaxWidth`.

Reply:
I changed this name to be tooltip-specific c513c65a457b25da7d52948b0e638924c7b4a637.

Any better suggestions are welcome 🙂